homeassistant/components/iqvia/sensor.py

```diff
@@ -105,6 +105,7 @@ class Rating(NamedTuple):
         key=TYPE_ALLERGY_TOMORROW,
         name="Allergy index: tomorrow",
         icon="mdi:flower",
+        state_class=SensorStateClass.MEASUREMENT,
```

Comment:
I think it doesn't make sense actually? It is not a current measurement?
It's a forecast/prediction.

From our developer documentation: 

> The state represents a measurement in present time, not a historical aggregation such as statistics or a prediction of the future.

<https://developers.home-assistant.io/docs/core/entity/sensor#available-state-classes>

Reply:
The sensor is a device that gives the current measurement prediction... If this isn't the correct state_class, then what is?

Reply:
None, as it is a future prediction.

Reply:
So is this a recorder bug then? I'm getting errors saying this sensor has an invalid state class.

Reply:
> So is this a recorder bug then? I'm getting errors saying this sensor has an invalid state class.

It should not throw errors, but lets figure that out in an issue report instead 😉